test-cases/tests/installation/a30-validate-installation-of-rhoam-addon-and-integration-with-ld.md

```diff
@@ -86,6 +87,24 @@ watch "oc get rhmi rhoam -n redhat-rhoam-operator -o json | jq .status.stage"
 7. Go to `Monitoring` section of your cluster in OCM UI
    > Make sure no alerts are firing
 
+**Verify custom RHOAM CIDR range was applied correctly**
```

Comment:
Could we also write a case where there is no cidr value passed by the add on params that cro does provide a sensible default and the installation was successful?
It would be the same steps as this one except for not passing the cidr param and when checking in aws the cidr block may vary but should be within certain ranges listed here: https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Subnets.html#vpc-sizing-ipv4 and should not overlap with the cluster machine cidr. If it does overlap then cro would error out. If it doesn't find an option then cro would error out. so validation would be that there are no errors from cro.

Reply:
This scenario should be covered by installation pipeline - it [doesn't provide any addon params](https://github.com/integr8ly/delorean/blob/0cd8e05a49540c0c505c3c291629dd737d7cc818/scripts/ocm/ocm.sh#L144), so CIDR block has to be created by CRO. If the pipeline finishes successfully, it means that the CIDR block was correct. So I will add a note to this test case that this scenario is already covered by pipeline.